opensrp-anc/src/main/java/org/smartregister/anc/library/fragment/ProfileContactsFragment.java

```diff
@@ -56,6 +61,13 @@
     private ProfileFragmentContract.Presenter presenter;
     private String baseEntityId;
     private String contactNo;
+    private Button dueButton;
+    private ButtonAlertStatus buttonAlertStatus;
+    private HashMap<String, String> clientDetails;
+    private View noHealthRecordLayout;
+    private ScrollView profileContactsLayout;
+    private View statusButton;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Perhaps 'statusButton' could be replaced by a local variable.](https://app.codacy.com/app/OpenSRP/opensrp-client-anc/pullRequest?prid=4657615)